util/warnp.c

```diff
@@ -8,6 +8,7 @@
 
 static int initialized = 0;
 static char * name = NULL;
+static int syslog_use = 0;
```

Comment:
This code, by Yoda, was written?

Not sure why this isn't `use_syslog`.

Reply:
IMO, this looks messy:
```
static int use_syslog = 0;
static int syslog_priority = LOG_WARNING;e
```
Besides, `syslog_foo` is easier to recognize as "something to do with syslog", rather than `bar_syslog`.

But ok, I can go back to `use_syslog` if you're not a fan of short green aliens.

Reply:
renamed to `use_syslog`